examples/streamlit-mnist/app.py

```diff
@@ -0,0 +1,41 @@
+import os
+import numpy as np
+import cv2
+from tensorflow.keras.models import load_model
+import streamlit as st
+from streamlit_drawable_canvas import st_canvas
+
+MODEL_DIR = os.path.join(os.path.dirname(__file__), 'model')
+if not os.path.isdir(MODEL_DIR):
+    os.system('runipy train.ipynb')
```

Comment:
Can delete this part.

Reply:
Shall we add a license for this file?

Reply:
The original repo does not contain a license, thus I am not sure what license we should add.